CONTRIBUTING.md

```diff
@@ -78,30 +78,20 @@ det-deploy local fixture-down
 ### Starting manually
```

Comment:
Since we are deprecating det-deploy, I think we should just move this section above and maybe even remove the det-deploy.

The devops-ag is encouraging engineering to move towards using the tools dir over det-deploy.

Reply:
Sounds good.